registrations/utils.py

```diff
@@ -168,6 +168,15 @@ def move_waitlisted_to_attending(registration, count: int):
             registration.move_first_waitlisted_to_attending(first_on_list=first_on_list)
 
 
+def get_checkout_url_with_lang_param(checkout_url, lang_code):
```

Comment:
Type hints would be nice since this (most likely) assumes that the URL is `str`. Not sure about `lang_code`.
```suggestion
def get_checkout_url_with_lang_param(checkout_url: str, lang_code) -> str:
```